libsolidity/interface/FileReader.cpp

```diff
@@ -113,19 +113,23 @@ ReadCallback::Result FileReader::readFile(string const& _kind, string const& _so
 		vector<reference_wrapper<boost::filesystem::path>> prefixes = {m_basePath};
 		prefixes += (m_includePaths | ranges::to<vector<reference_wrapper<boost::filesystem::path>>>);
 
+		vector<boost::filesystem::path> searchedPaths;
```

Comment:
Now that we have this list, `candidates` could be converted to a list of references to `searchedPaths`. No need to store full copies.